src/Tribe/Repositories/Traits/Event.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Event trait that contains all of the ORM filters that can be used for any repository.
+ *
+ * @since TBD
+ *
+ * @package Tribe\Tickets\Repositories\Traits
+ */
+
+namespace Tribe\Tickets\Repositories\Traits;
+
+use Tribe__Date_Utils as Dates;
+use Tribe__Timezones as Timezones;
+
+/**
+ * Class Event
+ *
+ * @since TBD
+ */
+trait Event {
+
+	/**
+	 * Filters events whose end date occurs on or before the provided date; fetch is not inclusive.
+	 *
+	 * @since 4.9
+	 *
+	 * @param string|DateTime|int $datetime A `strtotime` parse-able string, a DateTime object or
+	 *                                      a timestamp.
+	 * @param string|DateTimeZone $timezone A timezone string, UTC offset or DateTimeZone object;
+	 *                                      defaults to the site timezone; this parameter is ignored
+	 *                                      if the `$datetime` parameter is a DatTime object.
+	 *
+	 * @return array An array of arguments that should be added to the WP_Query object.
+	 */
```

Comment:
:no_entry_sign: Function return type is not void, but function has no return statement

<sub>hash: 0b4a5765ceab678257bd1259b15d491f</sub>